### ISSUE TITLE
Round 9 vs. Round 10 Missing Tests

### ISSUE THREAD

**Issue by msmith-techempower** [TechEmpower/FrameworkBenchmarks#1273]
I have run a preview round or two at this point and have gotten the numbers to stabilize somewhat: http://sprunge.us/FDWI

Now, when we visualize those results and compare against round 9, we notice that there are tests missing or incomplete. We need to work towards getting these tests (the ones that ran in r9) working again, or we need an explanation as to why they are no longer relevant.

The following are the lists of tests (by type) that are missing. I will attempt to update each one as we close out pull requests fixing them, or as explanations as to their disappearance are given.

The format is:
`frameworks/[lang]/[framework]#[test]`
If you wanted to run and test one, you would issue:
`toolset/run-tests.py --mode verify --test [test]`.
If you wanted to make changed to try and get `[test]` running, you would make them in:
`frameworks/[lang]/[framework]`

**JSON**
Round 9 (123 tests) vs. Preview (94 tests)
* frameworks/C/onion#onion #1306
* frameworks/CSharp/aspent#aspnet-mono #1287
* frameworks/CSharp/aspnet#aspnet-mono-jsonnet #1287
* frameworks/CSharp/aspnet#aspnet-mono-servicestack #1287
* frameworks/CSharp/evhttp-sharp#evhttp-sharp #1302
* frameworks/CSharp/nancy#nancy-libevent2
* frameworks/CSharp/nancy#nancy-mono #1304
* frameworks/CSharp/servicestack#servicestack-nginx-default #1303
* frameworks/CSharp/servicestack#servicestack-xsp-default
* ~~frameworks/JavaScript/express#express~~ #1294
* ~~frameworks/JavaScript/hapi#hapi~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs~~ #1296
* ~~frameworks/Lua/lapis#lapis~~ Resolved in master
* ~~frameworks/Perl/mojolicious#mojolicious~~ Runs in master
* ~~frameworks/Perl/plack#plack~~ #1305
* ~~frameworks/Perl/web-simple#web-simple~~ Runs in master
* ~~frameworks/PHP/cakephp#cake~~ #1281
* frameworks/PHP/hhvm#hhvm #1310
* frameworks/PHP/laravel#laravel
* ~~frameworks/PHP/php#php~~ #1285
* ~~frameworks/PHP/php-senthot#senthot~~ #1284
* ~~frameworks/PHP/php-symfony2#symfony2~~ #1293
* ~~frameworks/PHP/php-symfony2-stripped#symfony2-stripped~~ #1293
* ~~frameworks/Ruby/rack#rack-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/rack#rack-ruby~~ Renamed  #1041
* ~~frameworks/Ruby/rails#rails-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/rails#rails-ruby~~ Renamed  #1041
* ~~frameworks/Ruby/sinatra#sinatra-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/sinatra#sinatra-ruby~~ Renamed  #1041
* ~~ninja~~ Removed #800
* ~~play1~~ Removed #1015
* ~~play2~~ Removed #1015
* ~~play-scala~~ Removed #1024
* ~~frameworks/Python/tornado#tornado-pypy~~ 3685f761b No idea why this was removed, but it was

**DB**
Round 9 (144 tests) vs. Preview (102 tests)
* frameworks/C/onion#onion-raw #1306
* frameworks/CSharp/aspent#aspnet-mono-mongodb-raw #1287
* frameworks/CSharp/aspent#aspnet-mono-mysql-entityframework #1287
* frameworks/CSharp/aspent#aspnet-mono-mysql-raw #1287
* frameworks/CSharp/aspent#aspnet-mono-postgresql-entityframework #1287
* frameworks/CSharp/aspent#aspnet-mono-postgresql-raw #1287
* frameworks/CSharp/nancy#nancy-libevent2
* frameworks/CSharp/nancy#nancy-mono #1304
* frameworks/CSharp/servicestack#servicestack-nginx-mongodb #1303
* frameworks/CSharp/servicestack#servicestack-nginx-mysql #1303
* frameworks/CSharp/servicestack#servicestack-nginx-postgresql #1303
* frameworks/CSharp/servicestack#servicestack-xsp-mongodb
* frameworks/CSharp/servicestack#servicestack-xsp-mysql
* frameworks/CSharp/servicestack#servicestack-xsp-postgresql
* frameworks/Haskell/yesod#yesod-mongodb-raw
* frameworks/Java/wicket#wicket
* ~~frameworks/JavaScript/express#express-mongodb~~ #1294
* ~~frameworks/JavaScript/express#express-mysql~~ #1294
* ~~frameworks/JavaScript/hapi#hapi~~ #1296
* ~~frameworks/JavaScript/hapi#hapi-mysql~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mongodb~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mongodb-raw~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mysql~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mysql-raw~~ #1296
* ~~frameworks/JavaScript/ringojs#ringojs-raw~~ Works in master
* ~~frameworks/Lua/lapis#lapis~~ Resolved in master
* ~~frameworks/Perl/kelp#kelp-raw~~ Renamed
* ~~frameworks/Perl/mojolicious#mojolicious-raw~~ Part of default
* ~~frameworks/Perl/plack#plack~~ #1305
* ~~frameworks/Perl/web-simple#web-simple-raw~~ Runs in master
* ~~frameworks/PHP/cakephp#cake~~ #1281
* frameworks/PHP/hhvm#hhvm #1310
* ~~frameworks/PHP/php#php~~ #1285
* ~~frameworks/PHP/php#php-raw~~ #1285
* ~~frameworks/PHP/php-senthot#senthot~~ #1284
* ~~frameworks/PHP/php-symfony2#symfony2~~ #1293
* ~~frameworks/PHP/php-symfony2#symfony2-raw~~ #1293
* ~~frameworks/PHP/php-symfony2-stripped#symfony2-stripped~~ #1293
* ~~frameworks/PHP/php-symfony2-stripped#symfony2-stripped-raw~~ #1293
* ~~frameworks/Ruby/rails#rails-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/rails#rails-ruby~~ Renamed  #1041
* ~~frameworks/Ruby/sinatra#sinatra-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/sinatra#sinatra-ruby~~ Renamed  #1041
* ~~ninja~~ Removed #800
* ~~play1~~ Removed #1015
* ~~play2~~ Removed #1015
* ~~play-jpa~~ Removed #1024
* ~~play-scala~~ Removed #1024
* ~~play-slick~~ Removed #1024
* ~~frameworks/Python/tornado#tornado-pypy~~ 3685f761b No idea why this was removed, but it was

**QUERIES**
Round 9 (143 tests) vs. Preview (101 tests)
* frameworks/C/onion#onion-raw #1306
* frameworks/CSharp/aspent#aspnet-mono-mongodb-raw #1287
* frameworks/CSharp/aspent#aspnet-mono-mysql-entityframework #1287
* frameworks/CSharp/aspent#aspnet-mono-mysql-raw #1287
* frameworks/CSharp/aspent#aspnet-mono-postgresql-entityframework #1287
* frameworks/CSharp/aspent#aspnet-mono-postgresql-raw #1287
* frameworks/CSharp/nancy#nancy-libevent2
* frameworks/CSharp/nancy#nancy-mono #1304
* frameworks/CSharp/servicestack#servicestack-nginx-mongodb #1303
* frameworks/CSharp/servicestack#servicestack-nginx-mysql #1303
* frameworks/CSharp/servicestack#servicestack-nginx-postgresql #1303
* frameworks/CSharp/servicestack#servicestack-xsp-mongodb
* frameworks/CSharp/servicestack#servicestack-xsp-mysql
* frameworks/CSharp/servicestack#servicestack-xsp-postgresql
* frameworks/Haskell/yesod#yesod-mongodb-raw
* frameworks/Java/wicket#wicket
* ~~frameworks/JavaScript/express#express-mongodb~~ #1294
* ~~frameworks/JavaScript/express#express-mysql~~ #1294
* ~~frameworks/JavaScript/hapi#hapi-mongodb~~ #1296
* ~~frameworks/JavaScript/hapi#hapi-mysql~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mongodb~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mongodb-raw~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mysql~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mysql-raw~~ #1296
* ~~frameworks/Lua/lapis#lapis~~ Resolved in master
* ~~frameworks/Perl/kelp#kelp-raw~~ Renamed
* ~~frameworks/Perl/mojolicious#mojolicious-raw~~ Part of default
* ~~frameworks/Perl/web-simple#web-simple-raw~~ Runs in master
* ~~frameworks/PHP/cakephp#cake~~ #1281
* frameworks/PHP/hhvm#hhvm #1310
* ~~frameworks/PHP/php#php~~ #1285
* ~~frameworks/PHP/php#php-raw~~ #1285
* ~~frameworks/PHP/php-senthot#senthot~~ #1284
* ~~frameworks/PHP/php-symfony2#symfony2~~ #1293
* ~~frameworks/PHP/php-symfony2#symfony2-raw~~ #1293
* ~~frameworks/PHP/php-symfony2-stripped#symfony2-stripped~~ #1293
* ~~frameworks/PHP/php-symfony2-stripped#symfony2-stripped-raw~~ #1293
* ~~frameworks/Ruby/rails#rails-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/rails#rails-ruby~~ Renamed  #1041
* ~~frameworks/Ruby/sinatra#sinatra-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/sinatra#sinatra-ruby~~ Renamed  #1041
* ~~ninja~~ Removed #800
* ~~play1~~ Removed #1015
* ~~play2~~ Removed #1015
* ~~play-jpa~~ Removed #1024
* ~~play-scala~~ Removed #1024
* ~~play-slick~~ Removed #1024 
* ~~frameworks/Python/tornado#tornado-pypy~~ 3685f761b No idea why this was removed, but it was

**FORTUNE**
Round 9 (91 tests) vs. Preview (67 tests)
* frameworks/C/onion#onion-raw #1306
* frameworks/CSharp/aspent#aspnet-mono-mongodb-raw #1287
* frameworks/CSharp/aspent#aspnet-mono-mysql-entityframework #1287
* frameworks/CSharp/aspent#aspnet-mono-mysql-raw #1287
* frameworks/CSharp/aspent#aspnet-mono-postgresql-entityframework #1287
* frameworks/CSharp/aspent#aspnet-mono-postgresql-raw #1287
* frameworks/CSharp/servicestack#servicestack-nginx-mongodb #1303
* frameworks/CSharp/servicestack#servicestack-nginx-mysql #1303
* frameworks/CSharp/servicestack#servicestack-nginx-postgresql #1303
* frameworks/CSharp/servicestack#servicestack-xsp-mongodb
* frameworks/CSharp/servicestack#servicestack-xsp-mysql
* frameworks/CSharp/servicestack#servicestack-xsp-postgresql
* frameworks/Java/wicket#wicket
* ~~frameworks/JavaScript/express#express-mysql~~ #1294
* ~~frameworks/JavaScript/hapi#hapi-mysql~~ #1296
* frameworks/PHP/hhvm#hhvm #1310
* ~~frameworks/Lua/lapis#lapis~~ Resolved in master
* ~~frameworks/PHP/php#php-raw~~ #1285
* ~~frameworks/PHP/php-senthot#senthot~~ #1284
* frameworks/PHP/php-silex#silex
* ~~frameworks/PHP/php-symfony2#symfony2~~ #1293
* ~~frameworks/PHP/php-symfony2#symfony2-php-templates~~ #1293
* ~~frameworks/PHP/php-symfony2#symfony2-raw~~ #1293
* ~~frameworks/PHP/php-symfony2-stripped#symfony2-stripped~~ #1293
* ~~frameworks/PHP/php-symfony2-stripped#symfony2-stripped-php-templates~~ #1293
* ~~frameworks/PHP/php-symfony2-stripped#symfony2-strippped-raw~~ #1293
* ~~frameworks/Ruby/rails#rails-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/rails#rails-ruby~~ Renamed  #1041
* ~~ninja~~ Removed #800
* ~~play-scala~~ Removed #1024
* ~~play-slick~~ Removed #1024

**UPDATE**
Round 9 (78 tests) vs. Preview (58 tests)
* ~~frameworks/Python/bottle#bottle-mysql-raw~~ Works in master
* frameworks/CSharp/aspent#aspnet-mono-mongodb-raw #1287
* frameworks/CSharp/aspent#aspnet-mono-mysql-entityframework #1287
* frameworks/CSharp/aspent#aspnet-mono-mysql-raw #1287
* frameworks/CSharp/aspent#aspnet-mono-postgresql-entityframework #1287
* frameworks/CSharp/aspent#aspnet-mono-postgresql-raw #1287
* frameworks/CSharp/servicestack#servicestack-nginx-mongodb #1303
* frameworks/CSharp/servicestack#servicestack-nginx-mysql #1303
* frameworks/CSharp/servicestack#servicestack-nginx-postgresql #1303
* frameworks/CSharp/servicestack#servicestack-xsp-mongodb
* frameworks/CSharp/servicestack#servicestack-xsp-mysql
* frameworks/CSharp/servicestack#servicestack-xsp-postgresql
* frameworks/Java/wicket#wicket
* ~~frameworks/JavaScript/express#express-mongodb~~ #1294
* ~~frameworks/JavaScript/express#express-mysql~~ #1294
* ~~frameworks/JavaScript/hapi#hapi-mongodb~~ #1296
* ~~frameworks/JavaScript/hapi#hapi-mysql~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mongodb-raw~~ #1296
* ~~frameworks/JavaScript/nodejs#nodejs-mysql-raw~~ #1296
* ~~frameworks/Lua/lapis#lapis~~ Resolved in master
* frameworks/PHP/hhvm#hhvm #1310
* ~~frameworks/PHP/php#php-raw~~ #1285
* ~~frameworks/Ruby/rails#rails-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/rails#rails-ruby~~ Renamed  #1041
* ~~ninja~~ Removed #800
* ~~play-scala~~ Removed #1024
* ~~play-slick~~ Removed #1024

**PLAINTEXT**
Round 9 (68 tests) vs. Preview (58 tests)
* frameworks/CSharp/aspent#aspnet-mono #1287
* frameworks/CSharp/evhttp-sharp#evhttp-sharp #1302
* frameworks/CSharp/servicestack#servicestack-nginx-default #1303
* frameworks/CSharp/servicestack#servicestack-xsp-default
* ~~frameworks/Java/undertow-edge#undertow-edge~~ Works in master
* ~~frameworks/JavaScript/nodejs#nodejs~~ #1296
* ~~frameworks/Lua/lapis#lapis~~ Resolved in master
* ~~frameworks/PHP/cakephp#cake~~ #1281
* frameworks/PHP/hhvm#hhvm #1310
* ~~frameworks/PHP/php#php~~ #1285
* ~~frameworks/Ruby/rack#rack-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/rack#rack-ruby~~ Renamed  #1041
* ~~frameworks/Ruby/sinatra#sinatra-jruby~~ Renamed  #1041
* ~~frameworks/Ruby/sinatra#sinatra-ruby~~ Renamed  #1041
* ~~ninja~~ Removed #800
* ~~frameworks/Python/tornado#tornado-pypy~~ 3685f761b No idea why this was removed, but it was

**Comment by hamiltont** [comment]
php Laravel - #1323

**Comment by hamiltont** [comment]
wicket - #814

**Comment by hamiltont** [comment]
TLDR - Except `onion` and `yesod`, the only thing missing are the `CSharp` tests

**Comment by pdonald** [comment]
A newer version of mono has been released so there is no need to install snapshots (which disappear after a while) anymore.

**Comment by hamiltont** [comment]
@msmith-techempower Is this now safe to close?

**Comment by msmith-techempower** [comment]
Yes
